fedmsg.d/sync2jira.py

```diff
@@ -53,7 +53,6 @@
                 'token_auth': 'YOUR_JIRA_ACCESS_TOKEN',
             },
         },
-        'default_github_project_fields': {'storypoints': ('Estimate', 'customfield_12310243')},
         'map': {
             'github': {
                 'GITHUB_USERNAME/Demo_project': {'project': 'FACTORY', 'component': 'gitbz',
```

Comment:
Shouldn't there be a `'github_project_fields'` key in line 59, now?

Reply:
This is an unrelated issue. I think somebody has forgotten to change the `updates` when they changed the structure. And `github_project_fields` is optional, even if it is missing, that doesn't matter, because the rest of the code doesn't show a full example.  I can create an issue (and separate PR) to update this part. Let me know if that's okay.

Reply:
> I think somebody has forgotten to change the updates when they changed the structure.

Agreed.

> And `github_project_fields` is optional, even if it is missing, that doesn't matter, because the rest of the code doesn't show a full example.

But, that's part of my concern:  this _is_ as far as I can tell, _example_ code, and so completely omitting a feature like `github_project_fields` from it doesn't serve the customer well.

And, given that you're already making changes to the module, why not bring it up to date, given that it is trivial to do so?  (You've already pasted an example configuration into [a nearby comment](https://github.com/release-engineering/Sync2Jira/pull/216/files?w=0#r1816770782), why not just paste it into the module?)

> I can create an issue (and separate PR) to update this part. Let me know if that's okay.

I wouldn't block this PR solely on this issue; it just seems like creating an issue and a separate PR would be more effort (given that you're presumably already going to update this PR) than just making the requested change here.

Reply:
The current `issues` only has an Ellipsis (...) as a dummy value to indicate that there many possible values.
I can change it to `'issue_updates': [...]` or do you want me to include ass possible values (this may increase in future). After some searching, I found these many available values:
```
issue_updates=[
  'comments',
  'upstream_id',
  'title',
  'description',
  'github_markdown',
  'upstream_id',
  'url',
  {'transition': 'Closed'},
  {'assignee': {'overwrite': False}},
  'github_project_fields'
],
```
Please suggest.

Reply:
For now, I have changed it to `issue_updates` with a list containing an Ellipsis. Let me know, if all possible values needs to be included.

Reply:
Done. fd926afa5a5c00d29d0062717929887da89a6f6f